go.mod

```diff
@@ -1,13 +1,16 @@
 module github.com/pressly/goose/v3
 
-go 1.18
+go 1.20
 
 require (
 	github.com/ClickHouse/clickhouse-go/v2 v2.9.1
 	github.com/go-sql-driver/mysql v1.7.1
+	github.com/google/go-cmp v0.5.9
 	github.com/jackc/pgx/v5 v5.3.1
 	github.com/microsoft/go-mssqldb v0.21.0
 	github.com/ory/dockertest/v3 v3.10.0
+	github.com/stretchr/testify v1.8.2
+	github.com/testcontainers/testcontainers-go v0.19.0
```

Comment:
There's already ory/dockertest, is there a limitation of the existing one?

I don't have a preference for one or the other, but I do prefer consistency when possible so there's only one way to do something. Makes it easier to maintain and reduces the mental overhead.

Reply:
Yeah. I tried to use dockertest as well, however I had a hard time figuring out an easy way to mount local files into the container. I can try to look again to figure out if I can get away with just dockertest.

In my experience testcontainer generally has been faster and has a better API.

Reply:
Ok cool, outside the scope of this PR. But do you think we should switch over to that library instead?

Reply:
> outside the scope of this PR. But do you think we should switch over to that library instead?
Yeah, I think switching over to testcontainers can be an improvement. But it can be done separately.

Reply:
The tests use dockertest now.